### ISSUE TITLE
All contents 403 errors

### ISSUE THREAD

**Issue by fordiy** [yuliskov/SmartTube#3646]
22.36 and 22.37 versions. any contents are not playable and 403 errors.

**Comment by Metalneck** [comment]
Same here, both on stable release and beta.

**Comment by seppuko** [comment]
Same, are versions prior to 22.36 unaffected by this bug? Too bad there's no rollback option.

**Comment by JGoewie** [comment]
Same here, both on stable release and beta. went all the way back to 22.32 and still same error. bummer. liked this app.

**Comment by BlackSloth1** [comment]
Same here

**Comment by seancondev** [comment]
same

**Comment by Ketec** [comment]
Feels like google has started banning 3rd party apps?

If so, I guess no more youtube.

**Comment by drnesr** [comment]
Same, with Philips Android TV.

**Comment by guhill1** [comment]
Same here, android TV with latest stable version.

**Comment by Bandid0** [comment]
same

**Comment by jwin74** [comment]
> Same, are versions prior to 22.36 unaffected by this bug? Too bad there's no rollback option.

21.xx is also affected.

**Comment by masterwishx** [comment]
> 22.36 and 22.37 versions. any contents are not playable and 403 errors.

Lasted versions still not working?

**Comment by Itsjustahobby** [comment]
> > 22.36 and 22.37 versions. any contents are not playable and 403 errors.
> 
> Lasted versions still not working?

Nope. I know the comment isn't directed at me which I apologize for, simply with to add two cents to this conversation. I've tried previous versions such as 21 as well as other versions such as Beta. Logging in/out & Clearing cache + Data. None of these work.

**Comment by Patraputt** [comment]
Contents not playing, showing 403 error. Only live feeds working.

**Comment by openips** [comment]
same

**Comment by weedo078** [comment]
The problem may be related to this post from a YouTube employee:
https://support.google.com/youtube/thread/269521462/enforcement-on-third-party-apps?hl=en

**Comment by zyberick** [comment]
Live content only playable. 22.37 beta

**Comment by Hiylow** [comment]
Nothing working. Same for other apps like pipepipe and freetube. All suddenly not working. .....KHAN!!!!!.….sorry I mean YOUTUBE!!!!!!!!. Sorry it's the trek in me.

**Comment by TecnikOfficial** [comment]
> Nothing working. Same for other apps like pipepipe and freetube. All suddenly not working. .....KHAN!!!!!.….sorry I mean YOUTUBE!!!!!!!!. Sorry it's the trek in me.

REVANCED on mobile is working welp there is not revanced for tv version gonna wait for fix ... i rather wait then switching to ad infested official youtube app i m using [tvbro](https://github.com/truefedex/tv-bro) browser to watch youtube on tv ad free

**Comment by Ketec** [comment]
Well, if they started enforcing then it looks like no more youtube. Their ad ridden cesspool and horrible tv app are not an option.

Using adblock8ng browsers on TV with a remote is near impossible for navigation/controls.

**Comment by TecnikOfficial** [comment]
> Well, if they started enforcing then it looks like no more youtube. Their ad ridden cesspool and horrible tv app are not an option.
> 
> Using adblock8ng browsers on TV with a remote is near impossible for navigation/controls.

yes i know its not an replacement but it works just 2 more clicks needed in my case the browser tht came with stb had ad block and its well optimised to remote ....

**Comment by linuxstv** [comment]
update to the latest 22.38.

**Comment by Tenderfoot56** [comment]
For the last 2 days, July 8,9 2024, I had the same errors as others in this string, Source Error 403. I had gone as far as to uninstall and reinstall on 2 FireTV Devices yesterday to no avail.  Today, I turned on the TV, loaded the SmartTube app, tested and it failed again. I went to settings, selected the update and attempted to update, but the 2nd Gen FireTV Cube indicated that I had to give the "unknown app" permission to install at which point the FireTV admin app opened where all of the apps are listed. I scrolled down to the SmartTube app and toggled the Off to ON to allow the app to install and went back to the SmartTube application where it now allowed me to install the Version 22.38 update. Following the update, I restarted the application and now both recorded YouTube videos as well as live stream video is working perfectly. Thanks for fixing this problem. YouTube is really pretty much unwatchable without SmartTube!
